vite-plugins/generate-build-info.js

```diff
@@ -1,5 +1,27 @@
 import { execSync } from 'child_process'
 
+/**
+ * Javascriptifaction of
+ * https://github.com/geoadmin/web-mapviewer/blob/4826f4a05df9a5a12fc3eea2a902618800fc9425/buildspec.yml#L43-L44
+ * so that it also functions properly on a window10/11 machine in PowerShell (or Window terminal)
+ *
+ * Original cmd is git show-ref | grep "$(git rev-parse HEAD)" | awk '!/refs/tags// {print $2}' |
+ * head -1
+ *
+ * @return {String} The git branch without "origin/" or "refs/" portion
+ */
+function getGitBranchEvenWhenOnCI() {
```

Comment:
Would simply call it `getGitBranch`. In comment above I would add a comment that this function retrieve the current matching git branch even if we are on a detached HEAD.